imblearn/over_sampling/_smote.py

```diff
@@ -448,6 +450,9 @@ class SVMSMOTE(BaseSMOTE):
 
     SMOTENC : Over-sample using SMOTE for continuous and categorical features.
 
+    SMOTEN : Over-sample using the SMOTE variable specifically for categorical
```

Comment:
SMOTEN : Over-sample using the SMOTE variant specifically for nominal features only.